src/executors/protect_executor.rs

```diff
@@ -60,6 +59,13 @@ where
                 .context("Error getting gas price: {}")?;
         }
         action.tx.set_gas_price(bid_gas_price);
+        // set max_priority_fee_per_gas to be 0
+        let eip1559_tx = action.tx.as_eip1559_mut();
+        if let Some(eip1559_tx) = eip1559_tx {
+            eip1559_tx.max_priority_fee_per_gas = Some(U256::zero());
+        }
```

Comment:
want to log error in the else case?

Reply:
this is acutally not needed in this executor